sections/main-password-footer.liquid

```diff
@@ -3,71 +3,71 @@
+        <a href="{{ settings.social_pinterest_link }}" class="link list-social__link">
           {%- render 'icon-pinterest' -%}
           <span class="visually-hidden">{{ 'general.social.links.pinterest' | t }}</span>
         </a>
       </li>
     {%- endif -%}
     {%- if settings.social_instagram_link != blank -%}
       <li class="list-social__item">
-        <a href="{{ settings.social_instagram_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+        <a href="{{ settings.social_instagram_link }}" class="link list-social__link">
           {%- render 'icon-instagram' -%}
           <span class="visually-hidden">{{ 'general.social.links.instagram' | t }}</span>
         </a>
       </li>
     {%- endif -%}
     {%- if settings.social_tiktok_link != blank -%}
       <li class="list-social__item">
-        <a href="{{ settings.social_tiktok_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+        <a href="{{ settings.social_tiktok_link }}" class="link list-social__link">
           {%- render 'icon-tiktok' -%}
           <span class="visually-hidden">{{ 'general.social.links.tiktok' | t }}</span>
         </a>
       </li>
     {%- endif -%}
     {%- if settings.social_tumblr_link != blank -%}
       <li class="list-social__item">
-        <a href="{{ settings.social_tumblr_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+        <a href="{{ settings.social_tumblr_link }}" class="link list-social__link">
           {%- render 'icon-tumblr' -%}
           <span class="visually-hidden">{{ 'general.social.links.tumblr' | t }}</span>
         </a>
       </li>
     {%- endif -%}
     {%- if settings.social_snapchat_link != blank -%}
       <li class="list-social__item">
-        <a href="{{ settings.social_snapchat_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+        <a href="{{ settings.social_snapchat_link }}" class="link list-social__link">
           {%- render 'icon-snapchat' -%}
           <span class="visually-hidden">{{ 'general.social.links.snapchat' | t }}</span>
         </a>
       </li>
     {%- endif -%}
     {%- if settings.social_youtube_link != blank -%}
       <li class="list-social__item">
-        <a href="{{ settings.social_youtube_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+        <a href="{{ settings.social_youtube_link }}" class="link list-social__link">
           {%- render 'icon-youtube' -%}
           <span class="visually-hidden">{{ 'general.social.links.youtube' | t }}</span>
         </a>
       </li>
     {%- endif -%}
     {%- if settings.social_vimeo_link != blank -%}
       <li class="list-social__item">
-        <a href="{{ settings.social_vimeo_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+        <a href="{{ settings.social_vimeo_link }}" class="link list-social__link">
```

Comment:
```suggestion
        <a href="{{ settings.social_vimeo_link }}" class="list-social__link link">
```


Small nitpick: since you updated the class order on the other elements, what do you think of updating them here too

Reply:
Makes sense - approved :)